src/PlayerActor.cpp

```diff
@@ -0,0 +1,33 @@
+#include "PlayerActor.h"
+
+
+PlayerActor::PlayerActor(){
+  Position position = Position();
```

Comment:
put these variables into the header. position and velocity

Reply:
or actually in this case remove the type declaration before the variables since u already have it in the header.